responses/matchers.py

```diff
@@ -12,6 +12,18 @@
     JSONDecodeError = ValueError
 
 
+def _create_key_val_str(input_dict):
+    """
+    Returns string of format {'key': val, 'key2': val2}
+    :param input_dict: dictionary to transform
+    :return: (str) reformatted string
+    """
+    key_val_str = "{{{}}}".format(
+        ", ".join(["{}={}".format(key, val) for key, val in sorted(input_dict.items())])
```

Comment:
Should this recurse into values that are not scalars? or will native python formatting be ok?

Reply:
nice point, I added a recursive conversion. Now it is a Frankenstein instead of nice one liner :D

now message will be truly compatible between py2 and py3. But please, have a look at implementation, if needed, I will revert commit back

Reply:
@markstory, if got some time, can you please review

Reply:
let's just use `json.dumps`, I assume in none of the cases key in dict will be an integer, so we can use that method with `sort_keys=True`